Core/Src/nero.c

```diff
@@ -9,22 +9,35 @@
 #include "cerberus_conf.h"
 #include "monitor.h"
 #include "pedals.h"
+#include "string.h"
 
 static int8_t mph = 0;
 
 void send_nero_msg()
 {
-	uint8_t nero_index;
+	struct __attribute__((__packed__)) {
+		bool home_mode;
+		nero_menu_t nero_index;
+		uint8_t mph;
+		bool tsms;
+		float torque_lim_percentage;
```

Comment:
can we keep packed structs as primitive types (i.e. uint8_t, etc.), no bools  or enums pls.  Just cast appropriately.  One big thing is floats cannot be accurately represented on the CAN line (cool read: IEEE754) without causing issues.  Therefore what we do here is multiply by a factor, cast to uint, then calypso has a `format` key where you can specify `divideXXX` where X is that factor.  In this case you have two options:
- Multiply by 100 in cerb then divide by 100 in calypso since this float is 0 to 1. 
- Mutliply by 100 in cerb then know the data recieved is going to be 0 to 100 not 0 to 1 and thats OK.

Either way the float cannot be in the packed, this will be uint8_t.  Feel free to see things like DCL in shep `compute.c` and calypso's `bms.json` for how this is handled.